lib/StripeResource.js

```diff
@@ -258,7 +298,19 @@ StripeResource.prototype = {
       makeRequestWithData(null, utils.stringifyRequestData(data || {}));
     }
 
-    function makeRequest(apiVersion, headers) {
+    function retryRequest(requestFn, apiVersion, headers, requestRetries) {
+      requestRetries += 1;
+
+      return setTimeout(
+        requestFn,
+        self._getSleepTime(requestRetries) * 1000,
```

Comment:
nit, I would expect `_getSleepTime` to return ms, not s; mind doing the `* 1000` in its return statement?